driver/product/run_JGDAS_ATMOS_GEMPAK_META_NCDC_wcoss2.sh_00

```diff
@@ -0,0 +1,147 @@
+#!/bin/sh
+#PBS -N gdas_atmos_gempak_meta_ncdc_00
+#PBS -j oe
+#PBS -l place=vscatter,select=4:ncpus=32:mpiprocs=32:mem=5GB
+#PBS -q debug
+#PBS -l walltime=00:30:00
+#PBS -A GFS-DEV
+#PBS -V
+#PBS -W umask=022
+
+cd $PBS_O_WORKDIR
+
+export PDY=`date -u +%Y%m%d`
+export PDY=20210824
+# export PDY=20210922
+
+export PDY1=`expr $PDY - 1`
+
+export cyc=00
+export cycle=t${cyc}z
+
+export threads=1
+export MP_LABELIO=yes
+
+set -xa
+export PS4='$SECONDS + '
+date
+
+#############################################################
+# Specify GFS versions
+#############################################################
+export gfs_ver=v16.0.0
+
+#############################################################
+# Specify Run Driver location
+#############################################################
+export driver=/lfs/h2/emc/vpppg/noscrub/$USER/packages/gfs.$gfs_ver/driver
+
+#############################################################
+# Load module version on WCOSS2
+#############################################################
+. ${driver}/product/run.version
```

Comment:
A new run.ver file should exist in a new versions subfolder at the top level (`versions/run.ver`). Please update this line for that. Please also rename the run.version file you created to `run.ver` and move it up and into a new subfolder at the top level called `versions`. I'll be placing a new build.ver file in that new subfolder as well. See slide 9 of this slidedeck for what I'm referring to regarding the versions subfolder:
https://docs.google.com/presentation/d/15v-7rEM2CkJlEzwX4sE_qJd8DFJCgI726pzghIM_AwE/edit#slide=id.ge5cc65791d_0_31
Thanks!